packages/fxa-content-server/app/scripts/lib/router.js

```diff
@@ -683,10 +683,14 @@ Router = Router.extend({
       if (
         reactRouteGroups[routeGroup].routes.find((route) => routeName === route)
       ) {
+        const showBackboneOAuthRoute =
```

Comment:
Alternatively, we could have changed this:
```
    'oauth(/)': function () {
      this.createReactOrBackboneViewHandler('oauth', IndexView);
    },
```

To only show the Backbone view instead of calling `createReactOrBackboneViewHandler`, since we're pointing to Backbone at the Express-level and if it reaches this line it should be rendering the Backbone app for that route. However I thought that'd be confusing, and I think our server-routing and client-routing logic around the React stuff should match as well as it can.